docs/book/src/plugins/available-plugins.md

```diff
@@ -0,0 +1,25 @@
+# Available plugins
+
+This section describes the plugins supported and shipped in with the Kubebuilder project see: 
+- [go/v3](go-v3-plugin.md)
+- [go/v2](go-v2-plugin.md)
+<!-- Plugins are configured to execute one or more subcommands. The main subcommands are:
+- Init
+- Edit
+- Create API
+- Create Webhook -->
```

Comment:
A plugin can have 1 or many interfaces implemented. 
However, I think we do not need any of this info here. We are covering them in the Plugins section already.

Reply:
But I think you are trying to add the content of : https://github.com/kubernetes-sigs/kubebuilder/blob/master/VERSIONING.md#versioning ( wdyt about a new section versioning only for it? hen, we can add all from this doc add clean up it as well to keep the info centralized in a place?

Reply:
Sorry for the late response. So should I create a new subsection under plugins section with the name plugins versioning?

Reply:
I think that would be great 👍

Reply:
Okay